voicer-client/src/components/audio/AudioRecorder.jsx

```diff
@@ -1,52 +1,87 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { axiosWithAuth } from '../axiosWithAuth/axiosWithAuth'
+import { useInputControl } from '../../hooks/useInputControl'
+import { Button, Card, InputGroup, FormControl, FormGroup, Col, FormText, FormLabel } from "react-bootstrap"
 
-//As the req.body fields
-//File
-//Title
-//Description
-
-class ReactUploadImage extends React.Component {
-
-    constructor(props) {
-        super(props);
-        this.state ={
-            file: null
-        };
-        this.onFormSubmit = this.onFormSubmit.bind(this);
-        this.onChange = this.onChange.bind(this);
-    }
-    onFormSubmit(e){
-        e.preventDefault();
-        const formData = new FormData();
-        formData.append('file',this.state.file);
-        formData.append('title','This is a title');
-        formData.append('description','This is a description');
-        const config = {
-            headers: {
-                'content-type': 'multipart/form-data'
-            }
-        };
-        axiosWithAuth()
-        .post("/api/voice",formData,config)
-            .then((response) => {
-                alert("The file is successfully uploaded");
-            }).catch((error) => {
-        });
-    }
-    onChange(e) {
-        this.setState({file:e.target.files[0]});
-    }
-
-    render() {
-        return (
-            <form onSubmit={this.onFormSubmit}>
-                <h1>File Upload</h1>
-                <input type="file" name="myImage" onChange= {this.onChange} />
-                <button type="submit">Upload</button>
-            </form>
-        )
-    }
+const ReactUploadMedia = () => {
+
+	const [media, setMedia] = useState({
+		file: null
+	})
+	const titleInput = useInputControl('')
+	const descriptionInput = useInputControl('')
+
+	const doSubmit = (e) => {
+		e.preventDefault()
+		const formData = new FormData()
+		formData.append('file', media)
+		formData.append('title', "")
+		formData.append('description', "")
+		const config = {
+			headers: {
+				'content-type': 'multipart/form-data'
+			}
+		}
+		axiosWithAuth()
+		.post("/api/voice", formData, config)
+			.then(res => {
+				alert("File successfully uploaded")
```

Comment:
we'll be adding in a cogo-toast for this :) gotta get david's refresh loaded first